packages/client/components/AvatarListUser.tsx

```diff
@@ -83,6 +82,8 @@ const AvatarListUser = (props: Props) => {
         isAnimated={isAnimated}
         borderColor={borderColor}
         width={width}
+        isConnected={!!isConnected}
+        onClick={onClick}
```

Comment:
moved `onClick` to Avatar so it can change pointer style based on this

Reply:
I've changed the logic to only change the cursor if an onClick handler is specified.